public_html/src/userObject.php

```diff
@@ -56,13 +56,20 @@ public function __construct(array $vars, $fromDB){
 			$this->toolAdmin = 0;
 			$this->checkuser = 0;
 			$this->developer = 0;
-			$this->useSecure = isset($vars['useSecure']);
+			$this->useSecure = !isset($vars['useSecure']);

```

Comment:
I just thought about it, that if we rip it out completely, there should be no backwards compatibility issues. Lets do that with this pull request.